Packs/AzureSecurityCenter/README.md

```diff
@@ -11,3 +11,64 @@ Azure Security Center provides threat protection for data centers within both cl
 
 # License information
 Must be enabled on at least 1 Azure subscription.
+
+<~XSIAM>
+
+### This pack includes:
+- Log Normalization - XDM mapping.
+
+### Supported Event Types:
+- Security Alerts
+
+### Supported Timestamp Formats:
+- MMM DD YYYY HH:MM:SS (UTC)
+
+***
+
+## Data Collection
+
+Cortex XSIAM supports two methods to fetch alerts from Microsoft Defender for Cloud:
+- Microsoft Defender for Cloud collector (API Collection).
+- Azure Event Hub.
+
+### Microsoft Defender for Cloud collector 
+
+This collection method is more relevant in cases where the number of subscriptions is low, as each integration's instance refers to one subscription.
+
+For more information on how to configure this integration, please refer to the integration's docs:

```

Comment:
```suggestion
For more information on how to configure this integration, refer to the integration's docs:

```
I think it makes sense to move this to after the instructions for creating an instance. WDYT?